modules/gdscript/gdscript_parser.cpp

```diff
@@ -898,6 +898,10 @@ GDScriptParser::Node *GDScriptParser::_parse_expression(Node *p_parent, bool p_s
 
 		} else if (tokenizer->get_token() == GDScriptTokenizer::TK_PR_IS && tokenizer->get_token(1) == GDScriptTokenizer::TK_BUILT_IN_TYPE) {
 			// 'is' operator with built-in type
+			if (!expr) {
+				_set_error("No identifier provided for is operator");
```

Comment:
Maybe enclose "is" in single quotes? Otherwise might read as a broken sentence.
Probably worth adding a trailing dot too, though it doesn't seem used consistently in `_set_error` calls.

Reply:
@akien-mga with my latest force push I've updated the PR, commit message, and error message to include 'is' in brackets and made the error message more like the other `_set_error` error messages.